tests/test_dmlrunner.py

```diff
@@ -2,14 +2,18 @@
 
```

Comment:
We should add a test that tests a split where the user doesn't want to split and wants to train on all the data.